backend/src/services/functions/function.service.ts

```
@@ -305,27 +305,57 @@ export class FunctionService {
    * Delete a function
    */
   async deleteFunction(slug: string): Promise<boolean> {
+    const client: PoolClient = await this.getPool().connect();
+
     try {
-      const result = await this.getPool().query(
+      await client.query('BEGIN');
+
+      const result = await client.query(
         'DELETE FROM functions.definitions WHERE slug = $1',
```

ESLint

[error] Replace `⏎········'DELETE·FROM·functions.definitions·WHERE·slug·=·$1',` with `'DELETE·FROM·functions.definitions·WHERE·slug·=·$1',·[`

(prettier/prettier)